<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientSettingForm</class>
 <widget class="QGroupBox" name="ClientSettingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>TESTER</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>状態：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="StateLabel">
       <property name="text">
        <string>非接続</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ポート番号：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="PortSpinBox">
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>2009</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>接続名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="NameLabel">
       <property name="text">
        <string>不明</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>IP：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="IPLabel">
       <property name="text">
        <string>不明</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="ComboBox">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;「TCPユーザー」&lt;/p&gt;&lt;p&gt;クライアントと通信する一般的な形式です。&lt;/p&gt;&lt;p&gt;「自動くん」&lt;/p&gt;&lt;p&gt;移動を行わず、探索を繰り返すデバッグ用のクライアントです。&lt;/p&gt;&lt;p&gt;「ManualClient」&lt;/p&gt;&lt;p&gt;手動で操作するデバッグ用のクライアントです。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <item>
        <property name="text">
         <string>TCPユーザー</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>自動くん</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ManualClient</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ConnectButton">
       <property name="text">
        <string>待機開始</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ConnectButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>ClientSettingForm</receiver>
   <slot>ConnectionToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ClientSettingForm</receiver>
   <slot>ComboBoxChenged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ComboBoxChenged(QString)</slot>
  <slot>ConnectionToggled(bool)</slot>
 </slots>
</ui>
